File: 백준/Silver/26069. 붙임성 좋은 총총이/붙임성 좋은 총총이.java
import java.io.BufferedReader;
import java.io.IOException;
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt(); 
        sc.nextLine(); 
        Set<String> meet = new HashSet<>(); 

        for (int i = 0; i < n; i++) {
            String[] input = sc.nextLine().split(" ");
            String a = input[0];
            String b = input[1];

            
            if (a.equals("ChongChong") || b.equals("ChongChong")) {
                meet.add(a);
                meet.add(b);
            }
            if (meet.contains(a) || meet.contains(b)) {
                meet.add(a);
                meet.add(b);
            }
        }


        System.out.println(meet.size());
    }
}
    

File: 백준/Silver/26069. 붙임성 좋은 총총이/붙임성 좋은 총총이.py
import sys
input = sys.stdin.readline

n = int(input())
meet = set()

for _ in range(n):
    a,b = input().split()
    if a=='ChongChong' or b=='ChongChong':
        meet.add(a)
        meet.add(b)
    if a in meet or b in meet:
        meet.add(a)
        meet.add(b)
print(len(meet))